spotify_to_ytmusic/controllers.py

```diff
@@ -101,6 +102,21 @@ def remove(args):
     ytmusic = YTMusicTransfer()
     ytmusic.remove_playlists(args.pattern)
 
-
+def search(args):
+    spotify, ytmusic = _init()
+    track = spotify.getSingleTrack(args.link)
+    tracks = {
+        "name": track["name"],
+        "artist": track["artists"][0]["name"],
+        "duration": track["duration_ms"] / 1000,
+        "album": track['album']['name']
+    }
+    video_id = ytmusic.search_songs([tracks], use_cached=args.use_cached)
+    print(f"https://music.youtube.com/watch?v={video_id[0]}")   
+
+def cache_clear(args):
+    path = os.path.dirname(os.path.realpath(__file__)) + os.sep
```

Comment:
Please move this into the new Cache class and use `pathlib` instead of `os.path`